docker/README.md

```diff
@@ -446,6 +446,8 @@ For more information, see the docs for
 | `CAS_MONGODB_URI`                           | The MongoDB Connection STring for CAS; this will default to `FIFTYONE_DATABASE_URI`                                                                                                                                                                                            | No                        |
 | `FIFTYONE_APP_ALLOW_MEDIA_EXPORT`           | Set this to `"false"` if you want to disable media export options                                                                                                                                                                                                              | No                        |
 | `FIFTYONE_APP_ANONYMOUS_ANALYTICS_ENABLED`           | Controls whether anonymous analytics are captured for the teams application. Set to false to opt-out of anonymous analytics.                                                                                                                                                                                                              | No                        |
+| `FIFTYONE_APP_BANNER_COLOR`           | Global banner background color in App                                                                                                     | No                        |
```

Comment:
@voxel51/aloha-shirts do I still need this?

Reply:
running
```
pre-commit run --show-diff-on-failure --color=always --all-files
```

removed the additions I had to help/README.md for the two new environment variables. lmk